.github/pull_request_template/adoption_pull_request_template.md

```diff
@@ -1,7 +1,14 @@
-# Contributor Covenant Adoption
+# Project Name - Edit this line
 
-## Project name
-Your project name
+### Project Source Link
 
-## Link to code of conduct in repo or documentation
-[Code of conduct](url)
+https://github.com/git/git - Edit this line
+
+### Link to code of conduct in repo or documentation
+
+https://github.com/git/git/blob/master/CODE_OF_CONDUCT.md - Edit this link
+
+#### Checklist:
+
+- [ ] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
+- [ ] I have performed a self-review of submitted resource and its follows the guidelines of the project.
```

Comment:
I have reviewed the submitted resource and it follows the recommended adoption guidelines.